kitsune/sumo/static/sumo/less/main.less

```diff
@@ -2036,3 +2036,29 @@ input[type=button],
     }
   }
 }
+/* bug1441583 experiment-fxa-cta-topbar */
+#promotions {
+  background: #003eaa;
+  background: linear-gradient(148deg,#c7216a 7%,#fda41e 100%);
+  height: 28px;
+  color: #fff;
+  text-align: center;
+  font-size: 1.25em;
+  padding: 3px 0;
+  a {
+    color: #fff;
+    margin-left: 10px;
+  }
+  a:hover {
+    text-decoration: underline;
+  }
+  a:before {
+    content: "»\a0";
```

Comment:
Not a big deal, but we usually place this chevron after the link text.

Reply:
Yeah, I tried it both ways and this one looked nicer (to me) in the horizontal context.